editor/animation_track_editor.cpp

```diff
@@ -5701,6 +5702,7 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	timeline->set_zoom(zoom);
 
 	edit = memnew(MenuButton);
+	edit->set_shortcut_context(get_instance_id());
```

Comment:
I would make it still pass a Control*, to make API easier, but internally get the ID